kbd-mode.el

```diff
@@ -151,6 +153,193 @@ form `@MACRO-NAME' with `kbd-mode-variable-name-face'."
         (font-lock-add-keywords 'kbd-mode macro-regexp)
       (font-lock-remove-keywords 'kbd-mode macro-regexp))))
 
+;;; Interactive Functions
+
+;;;###autoload
+(defun kbd-mode-start-demo ()
+  "Try current configuration in a demo buffer.
+Use \\[kbd-mode-stop-demo] to stop current demo kmonad instance.
+If configuration file has errors, demo will not start
+instead, a compilation buffer will be shown."
+  (interactive)
+  (when (shell-command "ps -C kmonad")
+    (setq kbd-mode-had-kmonad t)
+    (kbd-mode--old-kmonad-suspend))
```

Comment:
This kills the old process in every case, even when there are compile errors and the users is then left without a config—not a good idea :)

Reply:
Yeah terrible idea :smirk:, I hope you don't find these from stucking in a unusable layout.
I think best idea is when calling `kbd-mode--old-kmonad-resume` in https://github.com/kmonad/kbd-mode/pull/7/files/f58c4d6f67eb116d2fd0437c1fffb2cb88d0eda1#diff-cf3235637f83e6e6523db0be286ac51ecd446c87c004df99f5dbbc7cc16375e5R278-R285

Reply:
And not adding any logic in `kbd-mode-start-demo`

Reply:
[334f8b4](https://github.com/kmonad/kbd-mode/pull/7/commits/334f8b4925f258c5d6e3434bb259b0c7c8062e25)
Fixes, also it loads more job to `kbd-mode--toggle-demo`.
> I'm still not 100% sold on it, 

I'm trying to protect the function from your wrath :smile: